examples/calendar/backend/services/users/controller.ts

```diff
@@ -0,0 +1,58 @@
+import { NextFunction, Request, Response } from "express";
+
+import { User } from "../../models/user.model";
+
+export const find = (req: Request, res: Response, next: NextFunction) => {
+  // If a query string ?publicAddress=... is given, then filter results
+  const whereClause =
+    req.query && req.query.publicAddress
+      ? {
+          where: { publicAddress: req.query.publicAddress },
+        }
+      : undefined;
+
+  return User.findAll(whereClause)
```

Comment:
why are we executing this if the previous condition returns undefined?

Reply:
Updated code.